eventbridge.tf

```diff
@@ -13,8 +28,9 @@ module "observe_firehose_eventbridge" {
 
   kinesis_firehose = module.observe_kinesis_firehose
   iam_name_prefix  = local.name_prefix
-  rules = [
-    aws_cloudwatch_event_rule.wildcard,
-  ]
+  rules = concat(
```

Comment:
Seems like a useless `concat` here but not important

Reply:
seems needed because of the count

Reply:
Ah, right you are. `flatten` is easier to understand for this purpose:

```tf
rules = flatten([
  aws_cloudwatch_event_rule.wildcard,
  aws_cloudwatch_event_rule.cloudtrail
])
```

This way, you don't have to wrap non-count resources in a list.